docs/src/app_dev/overview/splice_app_apis.rst

```diff
@@ -93,3 +108,25 @@ The token must be passed as an `OAuth2 Bearer token <https://datatracker.ietf.or
    Authorization: Bearer yourtoken
 
 Requests always operate on the primary party of the user specified as the subject in the token.
+
+Port Configuration
+-------------------
+
+All Splice apps expose HTTP APIs.
+The port they are exposed on is configured under ``admin-api.port``, e.g.,
```

Comment:
I suspect this might be slightly misleading as this is usually exposed through an ingress and the actual port doesn't matter all that much.